tldw_Server_API/tests/Writing/test_manuscript_analysis_integration.py

```
@@ -0,0 +1,402 @@
+    _project_id, _chapter_id, scene_id = _create_project_chapter_scene(client)
+    import tldw_Server_API.app.api.v1.endpoints.writing_manuscripts as writing_endpoint
+
+    monkeypatch.setattr(
+        writing_endpoint,
+        "get_provider_manager",
+        lambda: SimpleNamespace(providers=["openai"], primary_provider="openai"),
+    )
+    monkeypatch.setattr(
+        writing_endpoint,
+        "is_model_known_for_provider",
+        lambda provider, model: False if provider == "openai" and model == "bad-model" else True,
```

Ruff (0.15.9)

[warning] Use `not ...` instead of `False if ... else True`

Replace with `not ...`

(SIM211)